src/appComponents/sessions/SessionsController.js

```diff
@@ -78,10 +80,11 @@ sessionsModule.controller('SessionsController',
                     singleSession: session
                 };
                 $mdDialog.show({
-                    controller: 'SessionDialogController',
-                    templateUrl: 'appComponents/sessions/sessiondialog.html',
-                    parent: angular.element(document.body),
-                    targetEvent: event
+
+                    "controller": 'SessionDialogController',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-webapp/file/2424379902/issues/source?bid=3233857&fileBranchId=3238420#l84)